traffic_ops/app/db/migrations/2023012316280200_create_new_role.down.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+DELETE FROM public.role_capability rc WHERE rc.role_id='trouter';
+
+DELETE FROM public."role" r WHERE r.name='trouter';
```

Comment:
In order to be POSIX-compliant, every file should end with a newline.

![no newline](https://user-images.githubusercontent.com/11163823/215171075-8c1ec790-90ed-4215-82ba-727000dd6f5b.png)